hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -464,7 +464,7 @@ def check_duplicate_gids(reason=''):
     '''
     Return False if any duplicate group id exist in /etc/group file, else return True
     '''
-    gids = _execute_shell_command("cat /etc/group | cut -f3 -d\":\"").strip()
+    gids = _execute_shell_command("cat /etc/group | cut -f3 -d\":\"", python_shell=True).strip()
```

Comment:
useless use of cat... again, not a critique of the actual patches here, but cut takes a file argument, so cat doesn't need to be lifted from disk. Also, python could perform the cut and the file open without invoking the shell at all. Not requesting a  change, just annoyed by the form.

Reply:
Same as above -- could definitely be cleaned up.